cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskManager.java

```diff
@@ -4,111 +4,102 @@
+  private final Map<Integer, Integer> _inProgressInterBrokerReplicaMovementsByBrokerId;
+  private final Map<Integer, Integer> _inProgressIntraBrokerReplicaMovementsByBrokerId;
+  private final Set<TopicPartition> _inProgressPartitionsForInterBrokerMovement;
   private final ExecutionTaskTracker _executionTaskTracker;
-  private final Set<TopicPartition> _inProgressPartitions;
   private final ExecutionTaskPlanner _executionTaskPlanner;
-  private final int _defaultPartitionMovementConcurrency;
+  private final int _defaultInterBrokerPartitionMovementConcurrency;
+  private Integer _requestedInterBrokerPartitionMovementConcurrency;
+  private final int _defaultIntraBrokerPartitionMovementConcurrency;
+  private Integer _requestedIntraBrokerPartitionMovementConcurrency;
   private final int _defaultLeadershipMovementConcurrency;
-  private Integer _requestedPartitionMovementConcurrency;
   private Integer _requestedLeadershipMovementConcurrency;
   private final Set<Integer> _brokersToSkipConcurrencyCheck;
-  private final Time _time;
-  private volatile long _inExecutionDataToMove;
   private boolean _isKafkaAssignerMode;
 
-  private static final String REPLICA_ACTION = "replica-action";
-  private static final String LEADERSHIP_ACTION = "leadership-action";
-  private static final String IN_PROGRESS = "in-progress";
-  private static final String PENDING = "pending";
-  private static final String ABORTING = "aborting";
-  private static final String ABORTED = "aborted";
-  private static final String DEAD = "dead";
-  private static final String ONGOING_EXECUTION = "ongoing-execution";
-  private static final String KAFKA_ASSIGNER_MODE = "kafka_assigner";
-
-  private static final String GAUGE_REPLICA_ACTION_IN_PROGRESS = REPLICA_ACTION + "-" + IN_PROGRESS;
-  private static final String GAUGE_LEADERSHIP_ACTION_IN_PROGRESS = LEADERSHIP_ACTION + "-" + IN_PROGRESS;
-  private static final String GAUGE_REPLICA_ACTION_PENDING = REPLICA_ACTION + "-" + PENDING;
-  private static final String GAUGE_LEADERSHIP_ACTION_PENDING = LEADERSHIP_ACTION + "-" + PENDING;
-  private static final String GAUGE_REPLICA_ACTION_ABORTING = REPLICA_ACTION + "-" + ABORTING;
-  private static final String GAUGE_LEADERSHIP_ACTION_ABORTING = LEADERSHIP_ACTION + "-" + ABORTING;
-  private static final String GAUGE_REPLICA_ACTION_ABORTED = REPLICA_ACTION + "-" + ABORTED;
-  private static final String GAUGE_LEADERSHIP_ACTION_ABORTED = LEADERSHIP_ACTION + "-" + ABORTED;
-  private static final String GAUGE_REPLICA_ACTION_DEAD = REPLICA_ACTION + "-" + DEAD;
-  private static final String GAUGE_LEADERSHIP_ACTION_DEAD = LEADERSHIP_ACTION + "-" + DEAD;
-  private static final String GAUGE_ONGOING_EXECUTION_IN_KAFKA_ASSIGNER_MODE = ONGOING_EXECUTION + "-"  + KAFKA_ASSIGNER_MODE;
-  private static final String GAUGE_ONGOING_EXECUTION_IN_NON_KAFKA_ASSIGNER_MODE = ONGOING_EXECUTION + "-non-"  + KAFKA_ASSIGNER_MODE;
```

Comment:
I have difficulty in finding where we moved these gauges?, did we remove them? If so, we should not remove these sensors or rename them, as they are critical for tracking execution progress information.

Reply:
it is moved into [`ExecutionTaskTracker`](https://github.com/linkedin/cruise-control/blob/ceed1c2ba7862360ab9a5f6f5c76ff6373f65aa3/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskTracker.java#L73)